AmazonChimeSDKDemo/AmazonChimeSDKDemo/VideoModel.swift

```diff
@@ -276,12 +286,51 @@ class VideoModel: NSObject {
             }
         }
     }
+    
+    func addAllRemoteVideoSourceInCurrentPageExceptUserPausedVideos() {
+        var updatedSources:[RemoteVideoSource: VideoSubscriptionConfiguration] = [:]
+        for remoteVideoTileState in remoteVideoStatesInCurrentPage {
+            logger.info(msg: "Updating Remote Video Source in current page")
+            for (key, value) in remoteVideoSourceConfigurations {
+                if (String(remoteVideoTileState.1.attendeeId) == key.attendeeId) {
+                    logger.info(msg: "Subscribing to video source attendeeId: \(key.attendeeId)")
+                    updatedSources[key] = value
+                    
+                }
+            }
+        }
+        audioVideoFacade.updateVideoSourceSubscriptions(addedOrUpdated: updatedSources, removed: [])
+    }
 
     func pauseAllRemoteVideos() {
         for remoteVideoTileState in remoteVideoTileStates {
             audioVideoFacade.pauseRemoteVideoTile(tileId: remoteVideoTileState.0)
         }
     }
+    
+    func removeAllRemoteVideos() {
+        var remoteVideoSourcesInCurrPage: [RemoteVideoSource] = []
+        for remoteVideoTileState in remoteVideoTileStates {
```

Comment:
ditto
There are many places having the same logic, they should refactored into a helper function.

Reply:
`remoteVideoTileStates` can be upto 25 I believe. this implementation will help.